<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于深度学习的雷达/侦察系统效能评估软件</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="MainTitle">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;基于深度学习的雷达/侦察系统效能评估软件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>551</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_train">
     <attribute name="title">
      <string>指标属性</string>
     </attribute>
     <widget class="QLabel" name="label_indexName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>指标名称</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_dbn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>311</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>深度学习</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>训练学习速率</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lr">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_epoch">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>迭代次数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Batch大小</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_batch">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_nodeNum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>161</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>节点数量</string>
       </property>
       <widget class="QCheckBox" name="checkBox_lv1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>第一层</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_lv1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_lv2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>第二层</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_lv2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_lv3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>第三层</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_lv3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_selectTrain">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>选择样本数据</string>
       </property>
      </widget>
      <widget class="QPushButton" name="command_startTrain">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>开始训练</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_saveModel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>保存模型</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_loadModel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>180</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>加载模型</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>计算方法</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_indexType">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>指标类型</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_indexName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_indexType">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_weight">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>201</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>加权计算</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>选择指标</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_subIndex">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>权重</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_weight">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="push_saveWeight">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="push_saveCurrentindex">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_clearCurrentindex">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_checkIndex">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>一致性校验</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_test">
     <attribute name="title">
      <string>评估计算</string>
     </attribute>
     <widget class="QPushButton" name="push_selectEvaluate">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>选择评估数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="command_startEvaluate">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始评估</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_saveResult">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_result">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>291</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_index">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>311</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>指标体系</string>
    </property>
    <widget class="QTreeWidget" name="tree_index">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>权重</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否加权</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="push_saveIndex">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存指标体系</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_selectIndex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>导入指标体系</string>
     </property>
    </widget>
    <widget class="QPushButton" name="push_confirmindex">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
